### PR TITLE
OHFJIRA-70 : old gui cleanup

### PR DESCRIPTION
**Pull request by kkovarik**
Old GUI in web-admin module removal

#### CHANGES
* removed old gui resources (css, ftl templates etc...), controllers and configuration
* configured spring-security to work with new admin-console properly: removed all 3xx redirects, if not logged, unauthorized 401 is returned, if login fails then 403
* changed configuration of basic authentication for webservices, as it has not correctly worked in standalone mode (see comment in WebSecurityConfig)
* switched location of spring-boot maven plugin, -exec repackaged archive is now only in web-admin submodule

#### RELATED JIRA
* https://openhubframework.atlassian.net/browse/OHFJIRA-70

### CODE REVIEW

File: web-admin/src/main/java/org/openhubframework/openhub/web/common/WebProps.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
     public static final String MESSAGES_LIMIT = OpenHubPropertyConstants.PREFIX + "admin.console.messages.limit";
 
+    public static final String SESSION_CONCURRENCY_LIMIT = OpenHubPropertyConstants.PREFIX + "admin.console.session-concurrency.limit";
```
Comment (pjuza) [inline web-admin/src/main/java/org/openhubframework/openhub/web/common/WebProps.java:32]: I added on wiki (https://openhubframework.atlassian.net/wiki/spaces/OHF/pages/33599/OpenHub+configuration)

**Review by pjuza — APPROVED** [review]
Big step to finish 2.x version. Thanks.

**Comment by hanusto** [comment]
The fastest way how to increase code coverage - remove unused code lines :)

File: web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java [hunk 2]
```diff
@@ -1,138 +0,0 @@
...
-import org.springframework.util.StringUtils;
-import org.springframework.web.bind.WebDataBinder;
-import org.springframework.web.bind.annotation.InitBinder;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.util.UriUtils;
-
-import org.openhubframework.openhub.admin.services.log.LogEvent;
-import org.openhubframework.openhub.admin.services.log.LogParser;
```
Comment (hanusto) [inline web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java:41]: Please remove LogParser class, or maybe whole package _org.openhubframework.openhub.admin.services.log_

Comment (hanusto) [reply web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java:41]: And message report service

File: web-admin/src/main/java/org/openhubframework/openhub/admin/config/AdminSecurityConfig.java [hunk 3]
```diff
@@ -63,20 +84,18 @@
...
-
-        @Override
         protected void configure(HttpSecurity http) throws Exception {
+            Constraints.notNull(sessionConcurrencyLimit.getValue(), "the sessionConcurrencyLimit must be configured.");
+
             // @formatter:off
             http.csrf().disable() // HTTP with disabled CSRF
                     .authorizeRequests() //Authorize Request Configuration
-                        .antMatchers(WEB_URI_PREFIX + "homepage/**")
+                        .antMatchers(WEB_URI_PREFIX + "console/**")
```
Comment (hanusto) [inline web-admin/src/main/java/org/openhubframework/openhub/admin/config/AdminSecurityConfig.java:93]: Maybe we can reuse `spring.mvc.static-path-pattern` property.

**Review by hanusto — CHANGES_REQUESTED** [review]
Please try to remove log packages and unused classes.

File: web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java (hunk 2 above)
Comment (kkovarik) [reply web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java:41]: Removed, entire package & messageReportService as well.

File: web-admin/src/main/java/org/openhubframework/openhub/admin/config/AdminSecurityConfig.java (hunk 3 above)
Comment (kkovarik) [reply web-admin/src/main/java/org/openhubframework/openhub/admin/config/AdminSecurityConfig.java:93]: Don´t know, the WEB_URI_PREFIX has trailing slash /, and property now starts with slash as well... /console/**...

**Review by hanusto — APPROVED** [review]

**Pull request merged**